py-development.Rmd

```diff
@@ -437,8 +461,11 @@ import numpy as np
 
 np.mean([1, 2, 3])
 ```
+Using this syntax makes `numpy` not defined within that code. 
```

Comment:
```suggestion
Using this syntax makes the variable `numpy` not defined in the current namespace. 
```

Note to self that I am not sure if we have introduced namesspaces at this point, but maybe this is something to sort out during an edit round of the entire novice material.

Hmm